sphinxcontrib/confluencebuilder/translator.py

```diff
@@ -95,10 +96,19 @@ def depart_document(self, node):
                 self.builder.config.confluence_header_file)
             try:
                 with io.open(header_file, encoding='utf-8') as file:
-                    self.body_final += file.read() + self.nl
+                    header_template_data = file.read()
             except (IOError, OSError) as err:
                 self.warn('error reading file {}: {}'.format(header_file, err))
 
+            # if no data is supplied, the file is plain text
+            if self.builder.config.confluence_header_data is None:
+                header = header_template_data
```

Comment:
(same idea as other comment) `header_template_data` may be undefined.

Also applies to `footer_template_data`.

Reply:
Done